README.md

```diff
@@ -1,4 +1,5 @@
-# in-toto [![Build Status](https://travis-ci.org/in-toto/in-toto.svg?branch=develop)](https://travis-ci.org/in-toto/in-toto) [![Coverage Status](https://coveralls.io/repos/github/in-toto/in-toto/badge.svg?branch=develop)](https://coveralls.io/github/in-toto/in-toto?branch=develop) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1523/badge)](https://bestpractices.coreinfrastructure.org/projects/1523)
+# in-toto [![Build Status](https://travis-ci.org/in-toto/in-toto.svg?branch=develop)](https://travis-ci.org/in-toto/in-toto) [![Coverage Status](https://coveralls.io/repos/github/in-toto/in-toto/badge.svg?branch=develop)](https://coveralls.io/github/in-toto/in-toto?branch=develop) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1523/badge)](https://bestpractices.coreinfrastructure.org/projects/1523) [![Build status](https://ci.appveyor.com/api/projects/status/473rj06yqc4h3ai1?svg=true)](https://ci.appveyor.com/project/SantiagoTorres/in-toto)
+
```

Comment:
Minor nit: I think we should use an ["organizational" account](https://www.appveyor.com/docs/team-setup/#setting-up-appveyor-account-for-github-organization) `in-toto` for this so that the url would be `https://ci.appveyor.com/project/in-toto/in-toto`. What do you think?

Reply:
sounds good to me, let me set it up

Reply:
Let's merge then. :)